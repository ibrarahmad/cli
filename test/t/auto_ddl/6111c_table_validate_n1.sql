-- 6111c - Validate tables on n1

-- Prepared statement for spock.tables so that we can execute it frequently in the script below
PREPARE spocktab AS SELECT nspname, relname, set_name FROM spock.tables WHERE relname = $1 ORDER BY relid;

-- Validate sub_tx_table0
-- Expected: table does not exist
\d sub_tx_table0
EXECUTE spocktab('sub_tx_table0');

-- Validate sub_tx_table2
-- Expected: table does not exist
\d sub_tx_table2
EXECUTE spocktab('sub_tx_table2');

-- Validate sub_tx_table3
-- Expected: table does not exist
\d sub_tx_table3
EXECUTE spocktab('sub_tx_table3');

-- Validate sub_tx_table5, sub_tx_table5a, sub_tx_table5c, sub_tx_table5b should not exist
-- Expected: tables do not exist
\d sub_tx_table5
EXECUTE spocktab('sub_tx_table5');
\d sub_tx_table5a
EXECUTE spocktab('sub_tx_table5a');
\d sub_tx_table5b
EXECUTE spocktab('sub_tx_table5b'); -- should not exist
\d sub_tx_table5c
EXECUTE spocktab('sub_tx_table5c');

-- Validate table_ctas1
-- Expected: table does not exist
\d table_ctas1
EXECUTE spocktab('table_ctas1');

-- Validate table_ctas2
-- Expected: table does not exist
\d table_ctas2
EXECUTE spocktab('table_ctas2');

-- Validate table_ctas3
-- Expected: table does not exist
\d table_ctas3
EXECUTE spocktab('table_ctas3');

-- Validate table_ctas4
-- Expected: table does not exist
\d table_ctas4
EXECUTE spocktab('table_ctas4');

-- Validate table_ctas5
-- Expected: table does not exist
\d table_ctas5
EXECUTE spocktab('table_ctas5');

-- Validate table_ctas6
-- Expected: table does not exist
\d table_ctas6
EXECUTE spocktab('table_ctas6');

-- Validate table_si1
-- Expected: table does not exist
\d table_si1
EXECUTE spocktab('table_si1');

-- Validate table_si2
-- Expected: table does not exist
\d table_si2
EXECUTE spocktab('table_si2');

-- Validate table_si3
-- Expected: table does not exist
\d table_si3
EXECUTE spocktab('table_si3');

-- Validate table_si4
-- Expected: table does not exist
\d table_si4
EXECUTE spocktab('table_si4');

-- Validate table_si5
-- Expected: table does not exist
\d table_si5
EXECUTE spocktab('table_si5');

-- Validate table_l1
-- Expected: table does not exist
\d table_l1
EXECUTE spocktab('table_l1');

-- Validate table_l2
-- Expected: table does not exist
\d table_l2
EXECUTE spocktab('table_l2');

-- Validate table_l3
-- Expected: table does not exist
\d table_l3
EXECUTE spocktab('table_l3');

-- Validate table_l4
-- Expected: table does not exist
\d table_l4
EXECUTE spocktab('table_l4');

-- Validate table_l5
-- Expected: table does not exist
\d table_l5
EXECUTE spocktab('table_l5');
