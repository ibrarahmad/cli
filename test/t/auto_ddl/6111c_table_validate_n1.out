-- 6111c - Validate tables on n1
-- Prepared statement for spock.tables so that we can execute it frequently in the script below
PREPARE spocktab AS SELECT nspname, relname, set_name FROM spock.tables WHERE relname = $1 ORDER BY relid;
PREPARE
-- Validate sub_tx_table0
-- Expected: table does not exist
\d sub_tx_table0
Did not find any relation named "sub_tx_table0".
EXECUTE spocktab('sub_tx_table0');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table2
-- Expected: table does not exist
\d sub_tx_table2
Did not find any relation named "sub_tx_table2".
EXECUTE spocktab('sub_tx_table2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table3
-- Expected: table does not exist
\d sub_tx_table3
Did not find any relation named "sub_tx_table3".
EXECUTE spocktab('sub_tx_table3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate sub_tx_table5, sub_tx_table5a, sub_tx_table5c, sub_tx_table5b should not exist
-- Expected: tables do not exist
\d sub_tx_table5
Did not find any relation named "sub_tx_table5".
EXECUTE spocktab('sub_tx_table5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5a
Did not find any relation named "sub_tx_table5a".
EXECUTE spocktab('sub_tx_table5a');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5b
Did not find any relation named "sub_tx_table5b".
EXECUTE spocktab('sub_tx_table5b'); -- should not exist
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d sub_tx_table5c
Did not find any relation named "sub_tx_table5c".
EXECUTE spocktab('sub_tx_table5c');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas1
-- Expected: table does not exist
\d table_ctas1
Did not find any relation named "table_ctas1".
EXECUTE spocktab('table_ctas1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas2
-- Expected: table does not exist
\d table_ctas2
Did not find any relation named "table_ctas2".
EXECUTE spocktab('table_ctas2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas3
-- Expected: table does not exist
\d table_ctas3
Did not find any relation named "table_ctas3".
EXECUTE spocktab('table_ctas3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas4
-- Expected: table does not exist
\d table_ctas4
Did not find any relation named "table_ctas4".
EXECUTE spocktab('table_ctas4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas5
-- Expected: table does not exist
\d table_ctas5
Did not find any relation named "table_ctas5".
EXECUTE spocktab('table_ctas5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_ctas6
-- Expected: table does not exist
\d table_ctas6
Did not find any relation named "table_ctas6".
EXECUTE spocktab('table_ctas6');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si1
-- Expected: table does not exist
\d table_si1
Did not find any relation named "table_si1".
EXECUTE spocktab('table_si1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si2
-- Expected: table does not exist
\d table_si2
Did not find any relation named "table_si2".
EXECUTE spocktab('table_si2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si3
-- Expected: table does not exist
\d table_si3
Did not find any relation named "table_si3".
EXECUTE spocktab('table_si3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si4
-- Expected: table does not exist
\d table_si4
Did not find any relation named "table_si4".
EXECUTE spocktab('table_si4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_si5
-- Expected: table does not exist
\d table_si5
Did not find any relation named "table_si5".
EXECUTE spocktab('table_si5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l1
-- Expected: table does not exist
\d table_l1
Did not find any relation named "table_l1".
EXECUTE spocktab('table_l1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l2
-- Expected: table does not exist
\d table_l2
Did not find any relation named "table_l2".
EXECUTE spocktab('table_l2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l3
-- Expected: table does not exist
\d table_l3
Did not find any relation named "table_l3".
EXECUTE spocktab('table_l3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l4
-- Expected: table does not exist
\d table_l4
Did not find any relation named "table_l4".
EXECUTE spocktab('table_l4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

-- Validate table_l5
-- Expected: table does not exist
\d table_l5
Did not find any relation named "table_l5".
EXECUTE spocktab('table_l5');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

